components/shuttleInformation/index.js

```diff
@@ -0,0 +1,139 @@
+/* eslint-disable no-console */
```

Comment:
can some of these be disabled globally in eslintr.json?

Reply:
Yes, this is something to consider perhaps in the future as it is re-occurring